doskvol-tools/src/components/generator/streetGenerator/StreetGenerator.vue

```diff
@@ -1,8 +1,96 @@
 <script setup lang='ts'>
+import BaseRadioGroup from '../../common/baseRadio/BaseRadioGroup.vue'
+import {ref} from 'vue'
+import BaseButton from "@/components/common/BaseButton.vue";
+import BaseTable from "@/components/common/baseTable/BaseTable.vue";
+import type {RadioOptions} from "@/components/common/baseRadio/RadioOptions";
+import BaseCard from "@/components/common/BaseCard.vue";
+
+const commonness : RadioOptions[] = [
+  {
+    label: "Pospolity",
+    value: "common"
+  },
+  {
+    label: "Rzadki",
+    value: "rare"
+  }
+]
+const changes = ref("")
```

Comment:
`changes`?

Reply:
This is only for check component work. In next step I will create object like Street and I will bind it with props from this object